### DISCUSSION TITLE
Licht einschalten ohne Helligkeitswert

### DISCUSSION THREAD

**Discussion by mbrockeu** [brutella/hkknx-public#189] [Q&A]
Ich teste gerade eine kleine Installation mit LED Tunable White Spots.
Diese sind am Aktor so eingestellt, dass diese mit HCL Laufen und die Farbwerte/Helligkeit über den Aktor gesteuert werden.

In Homekit kann ich alle Werte auch soweit überschreiben. Dies führt auch dazu, dass beim Schalten der Lampe in HomeKit die Helligkeit auf 100% gestellt wird. Dies wird auch im Monitor mit zwei Telegrammen Ein und dann Helligkeit protokolliert.

Gibt es eine Möglichkeit nur das Ein Telegramm zu senden oder liegt das eher an HomeKit. Es wäre nämlich ganz schön das Licht nicht voll aufzudrehen.

**Comment by brutella** [comment]
Dazu gab es bereits Issue #161

Die Änderungen sind seit Version 2.2.0 verfügbar. Sollte also funktionieren, wenn der Dimmwert nach dem Einschalten von Aktor gesendet wird.

**Reply by mbrockeu** [reply]
Offenbar scheint es wirklich zu gehen. Ist nur verwirrend, weil die Home App 100% anzeigt. In den Telegrammen aber keine drin sind. Aber dass wird vermutlich jetzt rausgefiltert? Zumindest dauert es ein wenig bis der korrekte Wert gezogen wird.

**Reply by brutella** [reply]
Telegramme werden keine gefiltert. Aber ich hab da eine Vermutung, warum die Home-App immer noch die 100% angezeigt... 🤔
